lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart

```diff
@@ -150,21 +150,21 @@ class MailboxDashBoardView extends BaseMailboxDashBoardView {
         if (controller.isSelectionEnabled()) {
           return _buildListButtonTopBarSelection(context);
         } else {
-          return controller.isSearchActive()
+          return controller.searchController.isSearchActive()
```

Comment:
IMO, You can create a searchController object in the MailboxDashBoardView. 
Ex: `final searchController = Get.find<SearchController>();` for reusability, 
Avoid calling `controller.searchController` repeatedly which messes up the code